packages/wrangler/e2e/README.md

```diff
@@ -6,69 +6,140 @@ You can also run these tests locally, but you'll need access to the `8d783f274e1
 
 You can then run the e2e test suite with the following commands (run them in root of the repo):
 
-```sh
-rm -rf .tmp
-pnpm --filter wrangler deploy .tmp/wrangler
-rm .tmp/wrangler/templates/tsconfig.json
-CLOUDFLARE_ACCOUNT_ID=8d783f274e1f82dc46744c297b015a2f CLOUDFLARE_API_TOKEN=$CLOUDFLARE_TESTING_API_TOKEN WRANGLER="node --no-warnings $PWD/.tmp/wrangler/bin/wrangler.js" WRANGLER_IMPORT="$PWD/.tmp/wrangler/wrangler-dist/cli.js" pnpm --filter wrangler run test:e2e --retry 0
+```zsh
+pnpm i
+pnpm build
+CLOUDFLARE_ACCOUNT_ID=8d783f274e1f82dc46744c297b015a2f CLOUDFLARE_API_TOKEN=<cloudflare-testing-api-token> WRANGLER="node --no-warnings $PWD/.tmp/wrangler/bin/wrangler.js" WRANGLER_IMPORT="$PWD/.tmp/wrangler/wrangler-dist/cli.js" pnpm --filter wrangler run test:e2e --retry 0
```

Comment:
This no longer works, since you've remove the `pnpm deploy` step and `.tmp/wrangler` won't be generated

Reply:
Yes you're right! I forgot to update that bit. I have been happily running the e2e tests without deploying a standalone package. I'll update the command.